apps/condo/domains/user/utils/serverSchema/requestLimitHelpers.js

```diff
@@ -0,0 +1,25 @@
+const conf = require('@open-condo/config')
+const { GQLError } = require('@open-condo/keystone/errors')
+
+const { MAX_ANONYMOUS_REQUESTS_FOR_IP_BY_DAY } = require('@condo/domains/user/constants/common')
+const { GQL_ERRORS } = require('@condo/domains/user/constants/errors')
+const { RedisGuard } = require('@condo/domains/user/utils/serverSchema/guards')
+
+
+const REDIS_GUARD = new RedisGuard()
+const MAX_ANONYMOUS_REQUESTS_BY_DAY = Number(conf['MAX_ANONYMOUS_REQUESTS_FOR_IP_BY_DAY']) || MAX_ANONYMOUS_REQUESTS_FOR_IP_BY_DAY
+const IP_WHITE_LIST = conf.IP_WHITE_LIST ? JSON.parse(conf.IP_WHITE_LIST) : []
+
+const checkDayAnonymousRequestLimitCounters = async (context, prefix, rawIp) => {
```

Comment:
I'm not sure that guard by ip is sufficient. What if I just switch VPNs? Use different proxies?

Reply:
I'd have added smth with confirm phone action. So that even if I change IPs I would not be able to use this mutation more then N times

Reply:
Phone numbers are a bit harder to get :-(

Reply:
also added guard by phone